### ISSUE TITLE
No longer seeing build feedback in stdout (stderr?) after upgrade to 0.5.x 

### ISSUE THREAD

**Issue by patcoll** [thoughtbot/ember-cli-rails#333]
I used to see first and incremental build times in what I'm assuming was stdout in the 0.4.x versions, but no longer see that. Is there an option to re-enable?

Would also be willing to implement this if pointed in the right direction.

**Comment by seanpdoyle** [comment]
this was likely affected by https://github.com/thoughtbot/ember-cli-rails/commit/f911bf04f293ee2117f84acf115d93574964dee6.

I'm not sure what behavior you'd like to see. Would you mind elaborating?

**Comment by patcoll** [comment]
Absolutely. It previously seemed to pipe the output of ember build and watch, which was useful to know how quickly my builds were happening as I developed, helping me make decisions about how to streamline that process. 

Now I see no output from the ember commands. It does seem it has to do with the commit you mentioned. I'm not sure how much of an effort it is to restore previous behavior given the new code structure.

**Comment by kayakyakr** [comment]
+1 on this. There are a bunch of benefits to piping it to stdout for me, including:

1. I get to see recompilation errors immediately and any jshint comments as soon as the file is saved
2. It gives me confirmation that the ember build process is actually running. Spent 2 hours trying to figure out why the output wasn't showing before I realized I needed to check the log directory. 

While I can do this by tailing the ember log file, it's more convenient to have it in the development output, or at least have some confirmation that the ember build and watcher is up and running.
